app/models/apple/api.rb

```diff
@@ -204,7 +208,7 @@ def make_bridge_request(uri, body)
       # TODO: vary this with the bridge endpoint url
       use_ssl = false
 
-      Net::HTTP.start(uri.hostname, uri.port, use_ssl: use_ssl) do |http|
+      Net::HTTP.start(uri.hostname, uri.port, use_ssl: use_ssl, read_timeout: 10.minutes) do |http|
```

Comment:
That's a long timeout!  Could this potentially get stuck for some reason, and tie up the feeder worker(s)?

Reply:
Right now there is no worker/job integration generally, or publishing pipeline specifically.  But afaik the worker timeout would take precedence, right?

Reply:
Ah true.  Forgot about that worker timeout.